dicoogle/src/main/java/pt/ua/dicoogle/server/web/servlets/plugins/PluginsServlet.java

```diff
@@ -140,6 +140,60 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
         }
     }
 
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        List<String> pathParts = sanitizedSubpathParts(req);
+
+        // the format must be /<type>/<name>/[enable|disable]
```

Comment:
Doesn't `/<type>/<name>/(enable|disable)` convey the intent better? The action isn't optional. :)

Reply:
It does :+1: